Kernel/Arch/x86_64/Time/RTC.cpp

```diff
@@ -8,7 +8,7 @@
 #include <Kernel/Arch/x86_64/IO.h>
 #include <Kernel/Arch/x86_64/NonMaskableInterruptDisabler.h>
```

Comment:
not sure whether this should be also sorted and how exactly… (iʼd guess that `Kernel/Arch/x86_64/Interrupts/NonMaskableInterruptDisabler.h` should be good but maybe youʼd like to spread `Arch` stuff under specific “packages” instead? 😶)

Reply:
Seems to me that the current arrangement here is sufficiently fine.